Python/Product/TestAdapter.Executor/Pytest/PytestTestDiscoverer.cs

```diff
@@ -59,16 +59,16 @@ ITestCaseDiscoverySink discoverySink
 
             _logger = logger ?? throw new ArgumentNullException(nameof(logger));
 
-            var workspaceText = settings.IsWorkspace ? Strings.WorkspaceText : Strings.ProjectText;
-            LogInfo(Strings.PythonTestDiscovererStartedMessage.FormatUI(PythonConstants.PytestText, settings.ProjectName, workspaceText, settings.DiscoveryWaitTimeInSeconds));
+            //var workspaceText = settings.IsWorkspace ? Strings.WorkspaceText : Strings.ProjectText;
+            //LogInfo(Strings.PythonTestDiscovererStartedMessage.FormatUI(PythonConstants.PytestText, settings.ProjectName, workspaceText, settings.DiscoveryWaitTimeInSeconds));
 
             var env = InitializeEnvironment(sources, settings);
             var outputFilePath = Path.GetTempFileName();
             var arguments = GetArguments(sources, settings, outputFilePath);
 
-            LogInfo("cd " + settings.WorkingDirectory);
-            LogInfo("set " + settings.PathEnv + "=" + env[settings.PathEnv]);
-            LogInfo($"{settings.InterpreterPath} {string.Join(" ", arguments)}");
+            //LogInfo("cd " + settings.WorkingDirectory);
+            //LogInfo("set " + settings.PathEnv + "=" + env[settings.PathEnv]);
```

Comment:
i think we should add back logs

Reply:
@bschnurr But that will be printed out for every source, is that ok? 10 test containers will print 10 times.

Reply:
maybe instead of "info' there is a 'verbose' or diagnostic flag